tests/rptest/services/producer_swarm.py

```diff
@@ -79,6 +81,9 @@ def _additional_args(self) -> str:
         if self._messages_per_second_per_producer is not None:
             cmd += f" --messages-per-second {self._messages_per_second_per_producer}"
 
+        if self._message_period is not None:
+            cmd += f" --message-period {self._message_period}"
```

Comment:
This is mutually exclusive with the --mesages-per-second, right? We can assert they are not both set.

Reply:
Yeah, I do assert it within `client-swarm` itself, though it wouldn't hurt to assert it here as well.